docs/modules/servicebus.md

```diff
@@ -47,7 +48,106 @@ Binding returns the following connection details and shared credentials:
+| `lockDuration`      | `string` | ISO 8601 timespan duration of a peek-lock; that is, the amount of time that the message is locked for other receivers. The lock duration time window can range from 5 seconds to 5 minutes. For example, `PT2M30S` sets the lock duration time to 2 minutes 30 seconds. | N        | "PT30S"                                                      |
+
+##### Bind
+
+Returns a copy of one shared set of credentials.
+
+###### Binding Parameters
+
+This binding operation does not support any parameters.
+
+###### Credentials
+
+Binding returns the following connection details and shared credentials:
+
+| Field Name         | Type     | Description            |
+| ------------------ | -------- | ---------------------- |
+| `connectionString` | `string` | Connection string.     |
+| `primaryKey`       | `string` | Secret key (password). |
+| `queueURL`         | `string` | Queue URL              |
+
+##### Unbind
+
+Does nothing.
+
+##### Deprovision
+
+Deletes the Service Bus queue.
+
+
+
+### Service: azure-servicebus-topic
+
+| Plan Name | Description                     |
+| --------- | ------------------------------- |
+| `topic`   | New topic in existing namespace |
+
+#### Behaviors
+
+##### Provision
+
+Provisions a new topic in an existing namespace. 
+
+###### Provisioning Parameters
+
+| Parameter Name      | Type     | Description                                                  | Required | Default Value                                                |
+| ------------------- | -------- | ------------------------------------------------------------ | -------- | ------------------------------------------------------------ |
+| `parentAlias`       | `string` | Specifies the alias of the namespace in which the  topic should be provisioned. **Note**: the parent must be a service-bus-namespace instance with `standard` or `premium` plan. | Y        |                                                              |
+| `topicName`         | `string` | The name of the topic                                        | N        | If not provided, a random name will be generated as the topic name. |
+| `maxTopicSize`      | `int`    | The maximum size of the topic in megabytes, which is the size of memory allocated for the topic. | N        | 1024                                                         |
+| `messageTimeToLive` | `string` | ISO 8601 default message timespan to live value. This is the duration after which the message expires, starting from when the message is sent to Service Bus. This is the default value used when TimeToLive is not set on a message itself. For example, `PT276H13M14S` sets the message to expire in 11 day 12 hour 13 minute 14 seconds. | N        | "PT336H"                                                     |
+
+##### Bind
+
+Returns a copy of one shared set of credentials.
```

Comment:
Some here. And another question: do we create a subscription to the topic for each binding?